ksqldb-execution/src/main/java/io/confluent/ksql/execution/codegen/CodeGenSpec.java

```diff
@@ -101,9 +101,8 @@ public String getStructSchemaName(final CreateStructExpression createStructExpre
     private final Map<ColumnName, String> columnRefToName = new HashMap<>();
     private final ImmutableListMultimap.Builder<FunctionName, String> functionNameBuilder =
         ImmutableListMultimap.builder();
-    private final ImmutableMap.Builder<CreateStructExpression, String> structToSchemaName =
-        ImmutableMap.builder();
-
+    private final Map<CreateStructExpression, String> structToSchemaName =
+            new HashMap<CreateStructExpression, String>();
```

Comment:
this change seems to be leftover from a previous commit - please make sure to rebase from master and push only the most recent commit to this PR branch. (same comment about the files below)